SortingUtils.java

```diff
@@ -0,0 +1,18 @@
+
+public class SortingUtils {
+
+    public static int[] doInsertionSort(int[] input){
+        
+        int temp;
+        for (int i = 1; i < input.length; i++) {
+            for(int j = i ; j > 0 ; j--){
+                if(input[j] < input[j-1]){
+                    temp = input[j];
```

Comment:
Add some print outs so that you can see what comparisons are done during the call to this method.